tests/linearizability/linearizability_test.go

```diff
@@ -196,6 +204,18 @@ type trafficConfig struct {
 	traffic     Traffic
 }
 
+func validateEventsMatch(t *testing.T, ops [][]watchEvent) {
+	for i := 1; i < len(ops); i++ {
+		shorterLength := len(ops[0])
+		if len(ops[i]) < shorterLength {
```

Comment:
How about sorting ops by length DESC ? This way we could compare 2 longest streams .

Assuming we have watch streams of length: [0, 5, 7] we will not validate that 5 & 7 are matching in the current implementation.

Reply:
Good point. For simplicity I added comparing all streams.

Reply:
Nice catch, forgot to fix change this. Moved longest history to 0 index.